src/types/Types.ts

```diff
@@ -0,0 +1,29 @@
+
+export interface SearchRequest {
+  keyword: string;
+}
+
+export interface User {
+  userId: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  phoneNumber: string;
+  address: string;
+}
+
+export interface DataInputRequest {
```

Comment:
`DataInputRequest` should just be same as `User`

This is the input payload to DataInputApi:
```
{
  "userId": "asdfasdf",
  "firstName": "vaibhav",
  "lastName": "vasant",
  "email": "v12@gmail.com",
  "phoneNumber": "7633824597",
  "address": "None"
}
```